api/write/options.go

```diff
@@ -25,6 +25,8 @@ type Options struct {
 	precision time.Duration
 	// Whether to use GZip compression in requests. Default false
 	useGZip bool
+	// Point settings, default tags
+	pointSettings *PointSettings
```

Comment:
Wouldn't be better to simply use `defaultTags` herein. Precision (as a point setting) is also part of `Options`.

Reply:
I'm 100% for this. This was my first idea as well, but when I looked at Java and Js client I see default tags are part of PointSetting

Reply:
PointSettngs serve different purpose in JS, it is not an API configuration iface.